src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -114,6 +114,19 @@ internal override IEnumerable<AssociativeNode> BuildAst(List<AssociativeNode> in
     [AlsoKnownAs("Dynamo.Nodes.DoubleInput", "Dynamo.Nodes.dynDoubleInput", "DSCoreNodesUI.Input.DoubleInput")]
     public class DoubleInput : NodeModel
     {
+        private string numberType;
+
+        [JsonProperty("NumberType")]
+        public string NumberType
```

Comment:
@alfarok instead of making this a property with get and set why don't we make it like `NodeType` and just return the correct string depending on the type. So doubleInput will override this property, it will be very similar to `NodeType` property implementation.